src/core/link-to-dfn.js

```diff
@@ -80,6 +81,16 @@ export async function run(conf, doc, cb) {
         const dfn = titles[target.title][target.for];
         if (dfn[0].dataset.cite) {
           $ant[0].dataset.cite = dfn[0].dataset.cite;
+        } else if (
+          conf.xref &&
+          dfn[0].dataset.lt &&
+          dfn[0].dataset.lt.includes($ant[0].textContent.toLowerCase())
+        ) {
+          const lt = dfn[0].dataset.lt.split("|");
+          const txt = $ant[0].textContent.toLowerCase();
+          const key = dfn[0].textContent.toLowerCase();
+          $ant[0].classList.add("xref");
```

Comment:
we do make use of `data-dfn-type` elsewhere, maybe we can classify it as an "xref"?

Reply:
For searching, `.xref` would be faster I believe. But we can make use of `data-dfn-type` in other ways.